### DISCUSSION TITLE
Why Forem use scss and not framework like TailwindCSS?

### DISCUSSION THREAD

**Discussion by AxelBriche** [forem/forem#18249] [Q&A]
Why Forem use scss and not framework like TailwindCSS?

Why have you named things with "crayon"?

**Comment by okyanusoz** [comment]
> Why have you named things with "crayon"?

Crayons is Forem's design system. They built their own design system. See https://storybook.dev.to

> Why Forem use scss and not framework like TailwindCSS?

1. Not everyone likes Tailwind. It's like Bootstrap in that it's a CSS framework. And, it produces crazy-looking HTML.

2. They could have built the website with TailwindCSS but they would be dependent on TailwindCSS. It's like marrying. If they decided to migrate to SCSS in the future, then it would be hard (they would have to go through a "divorce").

3. It's probably too late now. They could migrate to TailwindCSS, but migrating big projects is hard. Imagine migrating thousands of lines to Tailwind CSS.
